spring-cloud-gcp-starters/spring-cloud-gcp-starter-security-iap/pom.xml

```diff
@@ -20,8 +20,16 @@
         </dependency>
 
         <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-oauth2-resource-server</artifactId>
+            <groupId>org.springframework.security</groupId>
```

Comment:
Huh? We have just discussed that it is OK to use starters as dependencies in starters.

Reply:
Might as well be consistent with autoconfig. And, I guess, for someone trying to troubleshoot why the starter works but their application does not, it would be nice to see all the dependencies together and not have to do a recursive hunt for them.

Reply:
OK. Let it be. I just wanted to follow the practice from Spring Boot.
Won't hart to have ti like this...